library-sync.sh

```diff
@@ -3,6 +3,71 @@
 # utility script to gather template/imagestream content from https://github.com/openshift/library
 # and store it in this repo (cannot access other repos with dist git, and advised against git submodules
 
+# Without any commandline arguments, this script only updates the OCP supported samples. This behavior
+# can be modified using these commandline arguments:
+# * --okd - the OKD samples are updated
+# * --ocp - the OCP supported samples are updated
+# * --ocp-all - all OCP samples are updated including the unsupported ones
```

Comment:
minor thing - I would include these descriptions in a `-h` message or something like that, if the script is meant also for local usage as I believe.